packages/turbo-gen/src/commands/raw/index.ts

```diff
@@ -1,59 +1,73 @@
+import { convertCase, logger } from "@turbo/utils";
```

Comment:
This is the file responsible for the command that `turbo` core uses to pass arguments to this package.

This entire file was not type-safe at all. It's a tricky one since we could get total garbage into the JSON argument here, but this re-works things so that it's a bit more defensive and happy. 

Good news is we had great test coverage for this so it's easy to validate.